src/diffusers/pipelines/flux/pipeline_flux.py

```diff
@@ -773,6 +790,9 @@ def __call__(
         lora_scale = (
             self.joint_attention_kwargs.get("scale", None) if self.joint_attention_kwargs is not None else None
         )
+        # should this not be like the following?
+        # has_neg_prompt = negative_prompt is not None or (negative_prompt_embeds is not None and negative_pooled_prompt_embeds is not None)
+        # do_true_cfg = true_cfg_scale > 1 and has_neg_prompt
```

Comment:
This is a question. Will discard once answered/resolved.

Reply:
Yes it should

Reply:
Thanks for confirming. I have removed it for now in this PR. Will open a separate PR with a test.